specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-02-01-preview/serviceGroups.json

```diff
@@ -0,0 +1,312 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service Group creation request is completed",
+            "schema": {
+              "$ref": "#/definitions/ServiceGroup"
+            }
+          },
+          "201": {
+            "description": "Service Group creation request initialized",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL for checking the ongoing status of the operation. To get the status of the asynchronous operation, send a GET request to the URL in Azure-AsyncOperation header value.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ServiceGroup"
+            }
+          },
+          "default": {
+            "description": "Error Response in making the request",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PutServiceGroup": {
+            "$ref": "./examples/ServiceGroup_Put.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "delete": {
+        "tags": [
+          "ServiceGroups"
+        ],
+        "operationId": "DeleteServiceGroup",
+        "description": "Delete a ServiceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ServiceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted response for delete request",
+            "headers": {
+              "Location": {
+                "description": "URL for determining when an operation has completed. Send a GET request to the URL in Location header.",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "No content response for delete request"
+          },
+          "default": {
+            "description": "Error Response in making the request",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeleteServiceGroup": {
+            "$ref": "./examples/ServiceGroup_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/providers/Microsoft.Management/serviceGroups/{serviceGroupName}/listAncestors": {
```

Comment:
>listAncestors [](http://example.com/codeflow?start=70&length=13)

Why using POST instead of GET?

Reply:
serviceGroups is the resourceType. This is an action of listing the ancestors of a given serviceGroup. 

This serves a collection response and a collection GET API path should end with resourceType, that matches the response items, which is not the case we have. 

so, opted for POST.